shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -559,6 +559,10 @@ - (void)dealloc {
   [super dealloc];
 }
 
+- (FlutterTextInputView*)textInputView {
```

Comment:
Please ensure that the signatures match. UIKit docs say that this needs to be `UIView*`. So lets keep that the same in both the header and implementation.